:bug: Missing Dockerfile path value on make build/build-no-cache

**Describe the bug**

Dockerfile path reference is missing in `make build` and `make build-no-cache, raising an error:

```text
$ make build
docker build -t geotrek . --build-arg BASE_IMAGE_TAG=
[+] Building 0.1s (1/1) FINISHED                                                                                   docker:default
 => [internal] load build definition from Dockerfile                                                                         0.0s
 => => transferring dockerfile: 2B                                                                                           0.0s
ERROR: failed to solve: failed to read dockerfile: open Dockerfile: no such file or directory
make: *** [Makefile:18 : build] Erreur 1
```

**To Reproduce**
Steps to reproduce the behavior:
1. Go to project root repository
2. Run on `make build or `make build-no-cache`

**Expected behavior**
Successful build 

**Screenshots**
Not required

--- a/Makefile
+++ b/Makefile
@@ -15,10 +15,10 @@ COLOR_DEBUG = \033[36m
 COLOR_RESET = \033[0m
 
 build:
-	docker build -t geotrek . --build-arg BASE_IMAGE_TAG=$(BASE_IMAGE_TAG)
+	docker build -t geotrek -f docker/Dockerfile . --build-arg BASE_IMAGE_TAG=$(BASE_IMAGE_TAG)
 
-build-no-cache:
-	docker build -t geotrek --no-cache .
+build_no_cache:
+	docker build -t geotrek -f docker/Dockerfile --no-cache .
 
 build_deb:
 	docker pull $(DISTRO)

--- a/docs/changelog.rst
+++ b/docs/changelog.rst
@@ -5,6 +5,9 @@ CHANGELOG
 2.110.0+dev     (XXXX-XX-XX)
 ----------------------------
 
+**Bug fixes**
+
+- Fix missing Dockerfile path on make build scripts
 
 
 2.110.0     (2024-11-13)

